plugins/out_s3/s3.c

```diff
@@ -1073,6 +1073,18 @@ static int upload_data(struct flb_s3 *ctx, struct s3_file *chunk,
         if (chunk) {
             s3_store_file_unlock(chunk);
             chunk->failures += 1;
+            char create_time_string[80];
+            strftime(create_time_string, 80, "%Y-%m-%d %H:%M:%S", localtime(&(create_time)));
```

Comment:
Fluent Bit code style prefers that you do not call one function within another.

Also, where does this `create_time` variable come from? The creation time should be obtained from the chunk structure.